OWNERS_ALIASES

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs at https://go.k8s.io/owners#owners_aliases
+
+aliases:
+  lwkd-maintainers:
+    - jberkus
+    - coderanger
+    - mbbroberg
```

Comment:
Are you sure these people are volunteering? Happy for more editors but want to make sure it's on purpose :D

Reply:
I don't know... can remove and add later, if needed too. :)

These folks were mentioned in the main issue body - https://github.com/kubernetes/org/issues/2589

Reply:
Aha, I trust Josh has spoken to them then. Or if not, easy to reverse (or not, I trust both of them to have admin access even if just as another back if both Josh and I are AFK).

Reply:
Matt Broberg volunteered when Contributor Comms approved taking LWKD.

Reply:
Bob is there as GH admin